hnn_core/tests/test_network.py

```diff
@@ -518,7 +636,7 @@ def test_network():
     # Test removing connections from net.connectivity
     # Needs to be updated if number of drives change in preceeding tests
     net.clear_connectivity()
-    assert len(net.connectivity) == 50
+    assert len(net.connectivity) == 4  # 2 drives x 4 target cell types
```

Comment:
sorry I don't get the comment ...

Reply:
Oops, this should be "2 drives x 2 target cell types".